.github/workflows/python-app.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+env:
+  NODE_VERSION: '10.x'                # set this to the node version to use
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+        node-version: ${{ env.NODE_VERSION }}
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
```

Comment:
Thanks for the comment. Indeed it will help to replicate same environment as CI I have abstracted the install, static analysis check, test etc in to python scripts. please have a review.